### PR TITLE
Fix dryrun

### PR DESCRIPTION
**Pull request by mlaszloffy**

### CODE REVIEW

File: pipedev-decider-utils/src/main/java/ca/on/oicr/pde/deciders/BasicDecider.java [hunk 1]
```diff
@@ -454,15 +454,19 @@ private ReturnValue launchWorkflows(Map<String, List<ReturnValue>> mappedFiles)
                                 continue;
                             }
 
+                            List<String> inputFileReport = new ArrayList<>();
+                            inputFileReport.add("Input file records:");
                             for (String line : studyReporterOutput) {
-                                LOGGER.debug(line);
+                                inputFileReport.add(line);
                             }
+                            LOGGER.info(inputFileReport.stream().collect(Collectors.joining("\n\n")));
```
Comment (apmasell) [inline pipedev-decider-utils/src/main/java/ca/on/oicr/pde/deciders/BasicDecider.java]: There's a prefix for `joining`, so you can go:
```
LOGGER.info(studyReporterOutput.stream().collect(Collectors.joining("\n\n", "Input file records:", "")));
```
and ditch the loop entirely.

Comment (mlaszloffy) [reply pipedev-decider-utils/src/main/java/ca/on/oicr/pde/deciders/BasicDecider.java]: Good call - fixed.

File: pipedev-common/pom.xml [hunk 2]
```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>ca.on.oicr.gsi</groupId>
         <artifactId>pipedev</artifactId>
-        <version>2.5</version>
+        <version>2.5.1-SNAPSHOT</version>
```
Comment (morgantaschuk) [inline pipedev-common/pom.xml]: you want to leave this as SNAPSHOT?

Comment (mlaszloffy) [reply pipedev-common/pom.xml]: Nope - I have updated the release version

**Pull request merged**